index.md

```diff
@@ -16,7 +16,7 @@ layout: main
 
 ### What should I replace it with?
 
-The frameworks of choice of the community are <a href="/topics/frameworks/#pkg-iron">iron</a>, <a href="/topics/frameworks/#pkg-conduit">conduit</a> and <a href="/topics/frameworks/#pkg-nickel">nickel</a>, best run with a <a href="/topics/database/#pkg-postgres">postgres</a> or <a href="/topics/database/#pkg-mysql">mysql</a> backend accompanied by <a href="/topics/database/#pkg-redis">redis</a> – all these drivers seem to be fairly mature. If you want to wrap that into an ORM <a href="/topics/database/#pkg-rustorm">rustorm</a> and <a href="/topics/database/#pkg-diesel">diesel</a> are at your disposal – both are still fairly young though.
+The frameworks of choice of the community are <a href="/topics/frameworks/#pkg-iron">iron</a>, <a href="/topics/frameworks/#pkg-conduit">conduit</a>, <a href="/topics/frameworks/#pkg-nickel">nickel</a> and <a href="/topics/frameworks/#pkg-rocket">rocket</a>, best run with a <a href="/topics/database/#pkg-postgres">postgres</a> or <a href="/topics/database/#pkg-mysql">mysql</a> backend accompanied by <a href="/topics/database/#pkg-redis">redis</a> – all these drivers seem to be fairly mature. If you want to wrap that into an ORM <a href="/topics/database/#pkg-rustorm">rustorm</a> and <a href="/topics/database/#pkg-diesel">diesel</a> are at your disposal – both are still fairly young though.
```

Comment:
No, I think we should list it, just mention that it's new and experimental and for now put it at the bottom of the list.